web/server/vue-cli/src/assets/userguide/userguide.md

```diff
@@ -317,6 +317,20 @@ stored.
 were reported by a checker that does not exists anymore because it was removed
 or renamed.
 
+The diagram below shows the detection status change which may happen when
+storing results to the server. For example if the detection status of a report
+in the database is **New** the server will change it to **Unresolved**,
+**Resolved**, **Off** or **Unavailable** based on the conditions above.
+
+![Detection status change](images/detection_status_flow_chart.png)
+
+*Notes*:
+- The detection status of a report is connected to the report id. It means that
```

Comment:
I think highlighting these sentences by moving them up after the first line `The detection status is the state of a bug report in a run.` ... would probably give more visibility.

Maybe one section about the difference between the detection status and the review status would also clarify what and when they are used or should be used. What do you think?